python/spec/dependabot/python/file_parser_spec.rb

```diff
@@ -65,6 +65,29 @@
       its(:length) { is_expected.to eq(3) }
     end
 
+    context "with jinja templates" do
+      let(:requirements_fixture_name) { "jinja_requirements.txt" }
+      describe "the first dependency" do
+        subject(:dependency) { dependencies.first }
+
+        it "has the right details" do
+          dependency = dependencies.first
```

Comment:
I believe you can just access the subject `dependency` without assigning here
```suggestion
```

Reply:
Good catch!